test/e2e/service.go

```diff
@@ -1151,6 +1171,31 @@ func testReachable(ip string, port int) bool {
 	return testReachableInTime(ip, port, podStartTimeout)
 }
 
+func testNetcatReachable(ip string, port int) {
+	con, err := net.Dial("udp", ip+":"+string(port))
+	if err != nil {
+		Failf("Failed to connect to: %s:%d (%s)", ip, port, err.Error())
+	}
+
+	_, err = con.Write([]byte("\n"))
+	if err != nil {
+		Failf("Failed to send newline: %s", err.Error())
+	}
+
+	var buf []byte = make([]byte, len("SUCCESS")+1)
```

Comment:
To Tim's point about reliability of the udp loadbalancer, can you confirm that a slightly heavier test runs as expected (throw some udp load at it via netcat, increase size of message, increase qps etc). You don't have to add it to the e2e, just check that it WAI. Since you mentione that you're using it already, that might be enough.